Added Cubone + Marowak and two of their attacks, Bone Club + Bonemerang

dangerstone opened the pull request:
I put a little recoil on Bonemerang, because I thought that would be fun. :)

dangerstone committed:
Added Cubone and Marowak as well as two of their attacks, Bone Club and Bonemerang

I put a little recoil on Bonemerang, because I thought that would be fun. :)

importhuman commented on attacklist.py:
```diff
@@ -154,7 +154,8 @@
 mudshot = Attack(name='mud shot', attCategory='ground', baseDamage=55, pokemonLevel=0, baseCount=15, recoil=0, heal=0, accuracy=0.95)
-
+boneclub = Attack(name='bone club', attCategory='ground', baseDamage=65, pokemonLevel=0, baseCount=20, recoil=0, heal=0, accuracy=0.85)
+bonemerang = Attack(name='bonemerang', attCategory='ground', baseDamage=50, pokemonLevel=43, baseCount=10, recoil=10, heal=0, accuracy=0.9)
```
pokemonLevel has been set to 0 for all attacks, I don't remember what its purpose was, just figured I should point that out. :)

dangerstone replied:
Oh, thank you! I'll correct this right away. :)

dangerstone committed:
Set pokemonLevel to 0

It was pointed out to me that all pokemonLevels are set to 0, so i corrected bonemerang

Sahil-k1509 commented on attacklist.py:
```diff
@@ -154,7 +154,8 @@
 mudshot = Attack(name='mud shot', attCategory='ground', baseDamage=55, pokemonLevel=0, baseCount=15, recoil=0, heal=0, accuracy=0.95)
-
+boneclub = Attack(name='bone club', attCategory='ground', baseDamage=65, pokemonLevel=0, baseCount=20, recoil=0, heal=0, accuracy=0.85)
+bonemerang = Attack(name='bonemerang', attCategory='ground', baseDamage=50, pokemonLevel=0, baseCount=10, recoil=10, heal=0, accuracy=0.9)
```
xD there is a small typo in boomerang spelling. 
edit: my bad. The attack is called bonemerang -_-

dangerstone replied:
I feel you, man, I kept typing "boomerang" instead because "bonemerang" seriously looks like a typo xD
